client/src/utils/dateParse.ts

```diff
@@ -0,0 +1,13 @@
+const appendZero = (number: number): string => {
+  return number < 10 ? `0${number}` : `${number}`;
```

Comment:
이런 방법도 있습니다~ 1자리 숫자가 들어오면 2자리까지 0으로 앞부터 채워주고 (4,'0')으로 했다면 0002 이런식으로 출력해줍니다~
```suggestion
  return number.padStart(2, '0');
  
```

Reply:
오 준범님이랑 같은 내용을 말씀하셨네요 ㅎㅎ
바꿨는데 반영이 안된거 같습니다 ㅎㅎ